actions/publish_pypi/action.yml

```diff
@@ -0,0 +1,118 @@
+name: "Build and publish the distribution to the PyPI server"
+description: "Build and publish PyPi wheel using tox for ML repos e.g. sparseml, compressed-tensors"
+
+inputs:
+  publish_pypi_internal:
+    description: "Publish the distribution to the internal PyPI server"
+    required: true
+
+  publish_pypi:
+    description: "Publish the distribution to the pypi.org"
+    required: true
+
+  built_type:
+    description: "Applies a set of rules depending on the environment. Available values: (dev|release|nightly|custom)"
+    required: true
+
+  custom_package_name:
```

Comment:
Either need to set to package_name and always expect from the user to override in the toml or we should auto load the current package name from the toml and append the suffixes with no defaulting of the package names to guidellm or anything else

Reply:
Just could be removed because of the `inject_build_props.py`